test/testReference.ts

```diff
@@ -26,6 +26,7 @@
 ///<reference path="components/selectionBoxLayerTests.ts" />
 
 ///<reference path="components/plots/plotTests.ts" />
+///<reference path="components/plots/XYPlotTests.ts" />
```

Comment:
xyPlotTests Let's stay with the convention that filenames should start with a lowercase letter